docs/extending-locust.rst

```diff
@@ -15,9 +15,10 @@ Here's an example on how to set up an event listener::
 
     from locust import events
     
-    @events.request_success.add_listener
-    def my_success_handler(request_type, name, response_time, response_length, **kw):
-        print("Successfully made a request to: %s" % name)
+    @events.request.add_listener
+    def my_request_handler(request_type, name, response_time, response_length, context, exception, **kw):
+        if not exception:
```

Comment:
now that it is a single handler it is worth showing both success and fail

(something like 
```
if exception:
    print(f"OMG, {name} failed with exception {exception}")
else:
    print(f"Successfully made a request {name}")
```